wordpress-mobile/WordPress-iOS
# Prototype Bottom Sheet integration

Addresses #10325.

### Testing

1. Review code.
1. Checkout branch and verify that tests pass.
1. For a `localDeveloper` build, navigate to _My Sites_ and tap **+**. 
1. Select `Demo bottom sheet`.
1. Interact with _Location_ search interface.
1. Tap _Cancel_ or _Save_ to dismiss.

### Out-of-Scope

- iPad orientation; `BlogListViewController` does not yet present this.
- Rotation between Portrait & Landscape.
- Integration with actual service.

### Video

![ezgif-5-f7830d8c8f7f](https://user-images.githubusercontent.com/221062/47873628-f4829480-ddce-11e8-9e9f-00bd1a99279f.gif)
